bundles/org.openhab.binding.smsmodem/README.md

```diff
@@ -9,6 +9,13 @@ The protocol stack is based on the no longer supported smslib project (more prec
 The following devices have been reported functional :
 
 - Huawei E180
+- Huawei E173
+- Huawei E1550
+- Huawei E272
+
+Those doesn't work:
```

Comment:
```suggestion
Devices that are reported to have issues:
```

Reply:
Oops, sorry for the bad english, and thanks for the fix.